questions/organization_and_scrum.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: post
+title: "Organization and Scrum"
+parent: Katas
+---
+The organization's management asks you what they should do for the Scrum Team. As a Scrum Master, what should you answer?
```

Comment:
```suggestion
The organization's management asks you what they should do for the Scrum Team. As a Scrum Master, what might you answer?
```

Reply:
I like the first version as I believe it provides more guidance and makes a good question (and I'd be very curious to hear the different answers to that 😄 )

Reply:
I'll reach out to Manolo on Monday and make sure he doesn't mind adjusting his question.